community/kernel/src/main/java/org/neo4j/kernel/api/KernelTransaction.java

```diff
@@ -157,6 +158,26 @@
      */
     Type transactionType();
 
+    /**
+     * Return transaction id if it was already assigned or {@link TransactionData#UNASSIGNED_TRANSACTION_ID} otherwise.
+     * Transaction id is assigned as soon as transaction committed.
+     * @return transaction id.
+     */
+    default long getTransactionId()
```

Comment:
@tinwelint do you know a place where i can check public/non public api? because it looks like i can't use package names for that then :)